packages/zcli-core/src/lib/request.ts

```diff
@@ -23,6 +23,7 @@ export const requestAPI = async (url: string, options: any = {}, json = false) =
 
   const authToken = await auth.getAuthorizationToken()
   const subdomain = process.env[EnvVars.SUBDOMAIN] || (await getSubdomain(auth))
+  const domain = process.env[EnvVars.SUBDOMAIN] ? (process.env[EnvVars.DOMAIN] || undefined) : await getDomain(auth)
```

Comment:
is `undefined` still required here?

Reply:
You are right, it is not required, if the env var is not defined it will return `undefined` as well